Opserver.Core/Data/Dashboard/Providers/WmiDataProvider.Polling.cs

```diff
@@ -310,6 +310,65 @@ FROM Win32_LogicalDisk
                 }
             }
 
+            private async Task GetServicesAsync()
+            {
+                if (!String.IsNullOrEmpty(ServicesPatternRegEx?.ToString()))
+                {
+                    string query = @"
+SELECT Caption,
+       Description,
+       DisplayName,
+       Name,
+       Started,
+       StartMode,
+       StartName,
+       State
+  FROM Win32_Service"; // windows services
+
+                    using (var q = Wmi.Query(Endpoint, query))
+                    {
+                        foreach (var service in await q.GetDynamicResultAsync().ConfigureAwait(false))
+                        {
+                            if (ServicesPatternRegEx.IsMatch(service.Name))
```

Comment:
`if (ServicesPatternRegEx?.IsMatch(service.Name) ?? true)` (with changes above of removing the outer `if`).